src/app/(BottomTabNavigation)/Profile/EditAddress/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { router } from 'expo-router';
+import React, { useEffect, useState } from 'react';
+import { Text, View, TextInput, TouchableOpacity } from 'react-native';
+
+import styles from './styles';
+import {
+  getCurrentUserInfo,
+  updateCurrUserAddress,
+} from '../../../../supabase/queries/auth';
+
+function EditNameScreen() {
+  const [currSession, setCurrSession] = useState<any>(null);
```

Comment:
Same comment as in `DeleteUser` but let's use `User` instead of `any` and set the default to `userInstance` which can be defined like `export const userInstance: User = {
  id: '',
  firstName: '',
  middleName: null,
  lastName: '',
  email: '',
  address: '',
};`!